Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticatorCoordinatorProxy.mm

```diff
@@ -387,6 +387,18 @@ static inline bool isCrossPlatformRequest(const Vector<AuthenticatorTransport>&
                 request.get().largeBlob.dataToWrite = WebCore::toNSData(*largeBlob->write).get();
         }
 
+        switch (options.userVerification) {
```

Comment:
```suggestion
request.get().userVerificationPreference = toASUserVerificationPreference(options.userVerification).get();
```

Reply:
I didn't realize we had this helper but this is much better, thanks!